### PR TITLE
K8SPS-55 replication related options

### PR DESCRIPTION
**Pull request by fiowro**

### CODE REVIEW

**Review by egegunes — APPROVED** [review]

File: source/architecture.rst [hunk 1]
```diff
@@ -8,7 +8,8 @@ Containers deployed with the Operator include the following components:
 
 * `Percona Server for MySQL <https://www.percona.com/doc/percona-server/LATEST/index.html>`_ (a free, fully compatible, enhanced, and open source drop-in replacement for any MySQL database),
 * `Percona XtraBackup <https://www.percona.com/doc/percona-xtrabackup/8.0/index.html>`_ (a hot backup utility for MySQL based servers that doesn’t lock your database during the backup),
-* `Orchestrator <https://github.com/openark/orchestrator>`_ (a replication topology manager for MySQL).
+* `Orchestrator <https://github.com/openark/orchestrator>`_ (a replication topology manager for MySQL, used when `asynchronous replication <https://dev.mysql.com/doc/refman/8.0/en/group-replication-primary-secondary-replication.html>`_ betweem MySQL instances :ref:`is turned on<mysql-clustertype>`),
```
Comment (tplavcic) [inline source/architecture.rst:11]: `betweem` -> `between`

File: source/architecture.rst [hunk 2]
```diff
@@ -8,7 +8,8 @@ Containers deployed with the Operator include the following components:
...
+* `MySQL Router <https://dev.mysql.com/doc/mysql-router/8.0/en/>`_, a proxy solution used when `group replication <https://dev.mysql.com/doc/refman/8.0/en/group-replication.html>`_ betweem MySQL instances :ref:`is turned on<mysql-clustertype>`).
```
Comment (tplavcic) [inline source/architecture.rst:12]: `betweem` -> `between`

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: source/operator.rst [hunk 3]
```diff
@@ -396,6 +560,36 @@ configuration options for the HAProxy service.
...
+|                                                                                                             |
++-----------------+-------------------------------------------------------------------------------------------+
+|                 | .. _orchestrator-affinity-advanced:                                                       |
+|                 |                                                                                           |
+| **Key**         | `orchestrator.affinity.advanced                                                           |
+|                 | <operator.html#orchestrator-affinity-advanced>`_                                          |
++-----------------+-------------------------------------------------------------------------------------------+
+| **Value**       | subdoc                                                                                    |
++-----------------+-------------------------------------------------------------------------------------------+
+| **Example**     |                                                                                           |
```
Comment (spron-in) [inline source/operator.rst:585]: just reading what we have here I don't understand what does this parameter do. Can you please provide an example?

File: source/operator.rst [hunk 4]
```diff
@@ -111,7 +125,35 @@ configuration options for the Percona Server for MySQL.
 | **Example**     | ``0``                                                                                     |
 +-----------------+-------------------------------------------------------------------------------------------+
 | **Description** | The number of the Percona Server for MySQL `semi-sync                                     |
-|                 | <https://dev.mysql.com/doc/refman/5.7/en/replication-semisync.html>`_ replicas            |
+|                 | <https://dev.mysql.com/doc/refman/8.0/en/replication-semisync.html>`_ replicas            |
++-----------------+-------------------------------------------------------------------------------------------+
+|                                                                                                             |
++-----------------+-------------------------------------------------------------------------------------------+
+|                 | .. _mysql-primaryServiceType:                                                             |
```
Comment (spron-in) [inline source/operator.rst:132]: What does it do? Can I configure other service parameters here? Like set annotations for this service? Is it for GR only?

**Review by spron-in — CHANGES_REQUESTED** [review]

File: source/operator.rst (hunk 4 above)
Comment (fiowro) [reply source/operator.rst:132]: thanks! I've added a note that these parameters are for asynchronous replication only

**Review by tplavcic — APPROVED** [review]

**Review by tplavcic — CHANGES_REQUESTED** [review]

**Pull request merged**